### PR TITLE
Update to README

### PR DESCRIPTION
**Pull request by mattshawoobe**
add specific blueprint content and remove some desktop.gov.au content which is no longer relevant

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,25 +83,18 @@ Content are Markdown files and normally end in `.md`. For help with Markdown syn
...
 
-Site search is provided by Algolia and is automatically integrated into deployed environments via CircleCI. If you are deploying to a local environment and need search, create a `_config-extras.yml` file in the root of your working directory substituting with correct values. Note that this file is not committed into the repository.
-
-```
-algolia:
-  application_id: "AAA"
-  index_name: "BBB"
-  search_only_api_key: "CCC"
-```
+Site search is provided by Luner Search and is available on the website.
```
Comment (SamBH) [inline README.md]: I believe it is "Lunr"

File: README.md [hunk 2]
```diff
@@ -97,25 +83,18 @@ Content are Markdown files and normally end in `.md`. For help with Markdown syn
...
-  search_only_api_key: "CCC"
-```
+Site search is provided by Luner Search and is available on the website. 
 
 ### Frequently asked questions
 
 #### What is this website built on?
 
-desktop.gov.au uses [Jekyll](https://jekyllrb.com/) to generate a static site and is hosted on [cloud.gov.au](https://cloud.gov.au/). It adheres to the [Jamstack](https://jamstack.org/) principles focussing on speed, security and hosting portability.
+The blueprint uses MkDocs desktop.gov.au uses [MkDocs](https://www.mkdocs.org/) with some custom CSS to generate a static site and is hosted by oobe.
```
Comment (SamBH) [inline README.md]: I think we need to delete the text in bold - "The blueprint uses **MkDocs desktop.gov.au uses** [MkDocs](https://www.mkdocs.org/) with some custom CSS to generate a static site and is hosted by oobe."

File: README.md [hunk 3]
```diff
@@ -97,25 +83,18 @@ Content are Markdown files and normally end in `.md`. For help with Markdown syn
...
 #### What is this website built on?
 
-desktop.gov.au uses [Jekyll](https://jekyllrb.com/) to generate a static site and is hosted on [cloud.gov.au](https://cloud.gov.au/). It adheres to the [Jamstack](https://jamstack.org/) principles focussing on speed, security and hosting portability.
+The blueprint uses MkDocs desktop.gov.au uses [MkDocs](https://www.mkdocs.org/) with some custom CSS to generate a static site and is hosted by oobe. 
 
-#### Can I contribute without a local development environment?
+#### Can I contribute?
 
-Yes of course, but you first need a Github account.
+Yes of course, you'll need a Github account.
```
Comment (SamBH) [inline README.md]: GitHub has a capital H

**Review by SamBH — CHANGES_REQUESTED** [review]
Minor feedback only

**Comment by mattshawoobe** [comment]
Updated all changes

**Review by mattshawoobe — COMMENTED** [review]
updated

**Review by SamBH — APPROVED** [review]
Review complete

**Pull request merged**